app/src/androidTest/java/com/android/sample/ui/workout/RunningScreenTest.kt

```diff
@@ -91,12 +93,57 @@ class RunningScreenTest {
 
     composeTestRule.onNodeWithTag("StartButton").performClick()
     composeTestRule.onNodeWithTag("PauseButton").performClick()
-    composeTestRule.onNodeWithTag("ResumeButton").performScrollTo().performClick()
-    composeTestRule.onNodeWithTag("PauseButton").assertExists()
+    composeTestRule.onNodeWithTag("ResumeButton").performClick()
+    composeTestRule.onNodeWithTag("PauseButton").assertIsDisplayed()
   }
 
+  //  @Test
+  //  fun finishButton_clickFinishesRunning() {
+  //    composeTestRule.setContent {
+  //      RunningScreen(
+  //          navigationActions = mockNavHostController,
+  //          runningWorkoutViewModel = mockRunningWorkoutViewModel)
+  //    }
+  //
+  //    composeTestRule.onNodeWithTag("StartButton").performClick()
+  //    composeTestRule.onNodeWithTag("PauseButton").performClick()
+  //    composeTestRule.onNodeWithTag("FinishButton").performClick()
+  //    composeTestRule.onNodeWithTag("FinishButton").assertExists()
+  //  }
+
+  //  @Test
+  //  fun saveToggleButton_isDisplayedWhenSaving() {
+  //    composeTestRule.setContent {
+  //      RunningScreen(
+  //          navigationActions = mockNavHostController,
+  //          runningWorkoutViewModel = mockRunningWorkoutViewModel)
+  //    }
+  //
+  //    composeTestRule.onNodeWithTag("StartButton").performClick()
+  //    composeTestRule.onNodeWithTag("PauseButton").performClick()
```

Comment:
Why are some tests commented? They should not be commented, or should not be here at all